requirements/dev.in

```diff
@@ -10,8 +10,8 @@ ipdb
 isort
 jsonschema~=2.5
 pep8
-pip==20.2.4
-pip-compile-multi~=2.3.0
+pip~=21.0
```

Comment:
This should be pinned to `==21.0.1` so it stays in line with the version in our docker container